Makefile.dep

```diff
@@ -316,3 +316,9 @@ ifneq (,$(filter cpp11-compat,$(USEMODULE)))
   USEMODULE += timex
   FEATURES_REQUIRED += cpp
 endif
+
+ifneq (,$(filter ng_pktbuf, $(USEMODULE)))
+  ifeq (,$(filter ng_pktbuf_dynamic, $(USEMODULE)))
```

Comment:
Yes, there is, right? If I would take out the inner guard, `ng_pktbuf_default` would be included everytime `ng_pktbuf` is defined. `USEMODULE+=ng_pktbuf_dynamic` would then have no effect...